modules/light-clients/08-wasm/types/wasm.go

```diff
@@ -1,61 +1,40 @@
 package types
 
 import (
-	"bytes"
-	"slices"
-
-	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/internal/ibcwasm"
 )
 
-// GetCodeHashes returns all the code hashes stored.
-func GetCodeHashes(ctx sdk.Context, cdc codec.BinaryCodec) (CodeHashes, error) {
-	wasmStoreKey := ibcwasm.GetWasmStoreKey()
-	store := ctx.KVStore(wasmStoreKey)
-	bz := store.Get([]byte(KeyCodeHashes))
-	if len(bz) == 0 {
-		return CodeHashes{}, nil
-	}
+// GetAllCodeHashes is a helper to get all code hashes from the store.
+// It returns an empty slice if no code hashes are found
+func GetAllCodeHashes(ctx sdk.Context) ([][]byte, error) {
```

Comment:
would a type alias for `[][]byte` be something people like? (though it would need tweaking of `ibcwasm.Codehashes` as a name.

Reply:
I prefer `[][]byte` tbh. But I'd like to hear what others think.

Reply:
imo i think the empty byte slice is fine for me, not sure it would add so much to alias it as it would probably be something like `emptyByteSlice` which would then be a bit self explanatory -- but also not too opinionated if others prefer the alias <3

Reply:
could also alias `[]byte` as `CodeHash`? `[]CodeHash`?

Reply:
My vote would also be for aliasing `[]byte` as `CodeHash` and using `[]CodeHash` as the return value. 

Also fine leaving as is.

Reply:
Can we create an issue for this? As I think this'd affect other functions not covered in this PR

Reply:
https://github.com/cosmos/ibc-go/issues/4993